src/View/Helper/FormHelper.php

```diff
@@ -313,11 +323,20 @@ public function control($fieldName, array $options = [])
                     $options['label'] = $this->injectClasses('custom-control-label', (array)$options['label']);
                     $options = $this->injectClasses('custom-control-input', $options);
 
-                    if ($this->_align === 'horizontal') {
-                        $options['templates']['checkboxFormGroup'] = $this->templater()->get('customCheckboxFormGroup');
+                    if (!$switch) {
```

Comment:
You can swap the blocks and avoid the negation. Same for all other `if` / `else` blocks you have added.